tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -68,6 +68,8 @@ def test_import_edit_export_contact(self):
         self.assertEqual(self.sim_contact['tel'][0]['value'], self.contact.tel['value'])
 
     def tearDown(self):
-        self.data_layer.delete_sim_contact(self.sim_contact['id'])
+        for contact in self.data_layer.sim_contacts:
+            if contact['name'][0].startswith('Name'):
+                self.data_layer.delete_sim_contact(contact['id'])
```

Comment:
"A company might start", you say. So currently, we don't have any company or anyone that's using sim card contacts on their device that starts with 'Name' in it and use it for Gaia UI tests, right?
Can't we just check this in, leave it in for a week, so devices on the lab have these test contacts wiped from their sim cards, then make the adjustment to use 'GAIAUITEST_' afterwards?